cli/login.go

```diff
@@ -219,6 +230,10 @@ func login() *cobra.Command {
 			return nil
 		},
 	}
+	cmd.Flags().StringVarP(&email, "email", "e", "", "Specifies an email address to authenticate with.")
+	cmd.Flags().StringVarP(&username, "username", "u", "", "Specifies a username to authenticate with.")
+	cmd.Flags().StringVarP(&password, "password", "p", "", "Specifies a password to authenticate with.")
```

Comment:
Should we allow env variables as well for use-cases where you want to script the login part but don't want to expose the password via `ps`? (E.g. `develop.sh`, albeit not a super sensitive use-case.)

Reply:
Agreed! Exposing the password via `ps` is sketchy 🕵️